packages/mdc-icon-button/README.md

```diff
@@ -87,7 +87,9 @@ Then instantiate an `MDCIconButtonToggle` on the root element.
 ```
 
 ```js
-var toggleButton = new mdc.iconButton.MDCIconButtonToggle(document.getElementById('add-to-favorites'));
+import {MDCIconButtonToggle} from '@material/icon-button';
+const iconToggle = new MDCIconButtonToggle(querySelector('#add-to-favorites'));
```

Comment:
nit: querySelector -> document.querySelector

Also it would be nice if we use same selector as above:

`document.querySelector('.mdc-icon-button')`

Reply:
@abhiomkar Changes have been made